sqlmesh/utils/jinja.py

```diff
@@ -99,23 +99,25 @@ def _advance(self, times: int = 1) -> None:
         )
 
 
-def call_name(node: nodes.Expr) -> t.Tuple[str, ...]:
+def node_name(node: nodes.Expr) -> t.Tuple[str, ...]:
     if isinstance(node, nodes.Name):
         return (node.name,)
     if isinstance(node, nodes.Const):
-        return (f"'{node.value}'",)
+        return (node.value,)
```

Comment:
@izeigerman why were constants being quoted? All the tests pass if I don't quote, so I'm wondering what it was solving? source args are constants, so the quotes were messing up getting the source package and name. I can add them back in and work around them if the quotes are indeed needed.